src/server/game/Entities/Player/Player.cpp

```diff
@@ -27243,3 +27247,186 @@ uint32 Player::DoRandomRoll(uint32 minimum, uint32 maximum)
 
     return roll;
 }
+
+uint32 Player::PlayScene(uint32 sceneId, Position const* position /*= nullptr*/)
```

Comment:
It would be nice to just move all of scene related methods to its own file and class instead of further bloating Player